Update stop-ITs shell script

sanjaysrikakulam opened the pull request:
updates parsing and adds comments and output statements for logs

kysrpex reviewed (approved):
I assume it is working (I saw some removed jobs on Grafana). 

You may consider replacing `$TODAY` with `$(date +'%d-%m-%y %H:%M:%S')`, because if there are too many jobs I assume the timestamp will be off by a few seconds (`cat` and `head` ain't free in NFS).

bgruening commented:
I remove a few manually.

sanjaysrikakulam commented:
> I assume it is working (I saw some removed jobs on Grafana).
> 
> You may consider replacing `$TODAY` with `$(date +'%d-%m-%y %H:%M:%S')`, because if there are too many jobs I assume the timestamp will be off by a few seconds (`cat` and `head` ain't free in NFS, neither `/usr/local/bin/galaxy_jwd`).

I tested it locally on the maintenance node by commenting out the `condor_rm` command. 

I initially added that instead of `TODAY`, but then I moved it because we are not super keen on the exact time. I added it just so we have some timestamps on the day and the hour it was removed in case we need to look into logs.
